backend/project.clj

```diff
@@ -41,7 +41,7 @@
                  [org.immutant/web "2.1.10" :exclusions [ch.qos.logback/logback-classic]]
                  [org.postgresql/postgresql "42.2.5"]
                  [ragtime "0.7.2"]
-                 [raven-clj "1.5.2"]
+                 [raven-clj "1.6.0-alpha3"]
```

Comment:
It seems that this depenency was published 8 days ago (non alpha one)
https://github.com/sethtrain/raven-clj/releases/tag/1.6.0

Reply:
changed!